ansible/www-standalone/resources/config/nodejs.org

```diff
@@ -76,36 +75,37 @@ server {
     error_log /var/log/nginx/nodejs/nodejs.org-error.log;
 
     gzip on;
-    gzip_static on;
     gzip_disable "MSIE [1-6]\.";
     gzip_types text/plain text/css application/javascript text/xml application/xml application/xml+rss image/svg+xml;
-    
-    open_file_cache max=10000 inactive=300s;
+
+    open_file_cache max=100000 inactive=300s;
     open_file_cache_valid 120s;
     open_file_cache_min_uses 2;
     open_file_cache_errors off;
 
-    error_page 404 @localized_404;
+    set $lang en;
```

Comment:
Very minor nitpick, can this be moved down to be directly above the error_page directive, as they're related?